draft-packages/button/KaizenDraft/Button/decision-tokens.scss

```diff
@@ -0,0 +1,24 @@
+@import "~@kaizen/design-tokens/sass/color";
+
+// Zen theme
+$dt-color-background-active: $kz-color-wisteria-100;
```

Comment:
Naming of design tokens inspired by this blog: https://medium.com/eightshapes-llc/naming-tokens-in-design-systems-9e86c7444676

TL;DR
`${namespace}-${type}-${property}-${variant}-{state}`
dt = design token namespace for variable
type = token type (one of color, shadow, animation, spacing, typography)
property = CSS property
variant = primary/disabled/etc if applicable
state = hover, focused, disabled

Reply:
I've done this because I didn't want to mix theme styling in the button itself.

Reply:
I like this! Is `dt` as the namespace intended to differentiate it from `kz` as a namespace for the "public" facing tokens?